cmd/cmd.go

```diff
@@ -127,6 +128,15 @@ kubectl neat get -- svc -n default myservice --output json`,
 	},
 }
 
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print kubectl-neat version",
+	Long:  "Print the version of kubectl-neat",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Printf("kubectl-neat version: %s\n", Version)
```

Comment:
Hi @itaysk, hope you're well :)

I started off with this basic implementation that only prints the version. Let me know if you would like to expand it by displaying other information like git commit or build date, and then perhaps also introduce a `--short` flag.

Also note that I am currently ignoring the persistent `--output` flag, as I couldn't find a good way to disable a global flag from a specific command. The version can of course be printed as JSON or YAML if you don't find it an overkill.